packages/opencanary/data_stream/events/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,759 @@
+      tag: Rename TFTP FILENAME field to opencanary.tftp.filename
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_TFTP'
+      field: opencanary.logdata.FILENAME
+      target_field: opencanary.tftp.filename
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename TFTP MODE field to opencanary.tftp.node
+      tag: Rename TFTP MODE field to opencanary.tftp.node
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_TFTP'
+      field: opencanary.logdata.MODE
+      target_field: opencanary.tftp.node
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename TFTP OPCODE field to opencanary.tftp.opcode
+      tag: Rename TFTP OPCODE field to opencanary.tftp.opcode
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_TFTP'
+      field: opencanary.logdata.OPCODE
+      target_field: opencanary.tftp.opcode
+      ignore_missing: true
+      ignore_failure: true
+
+  # VNC Events
+  - rename:
+      description: Rename VNC 'VNC Client Response' field to opencanary.vnc.client_response
+      tag: Rename VNC 'VNC Client Response' field to opencanary.vnc.client_response
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_VNC'
+      field: 'opencanary.logdata.VNC Client Response'
+      target_field: opencanary.tftp.vnc.client_response
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename VNC 'VNC Password' field to opencanary.vnc.password
+      tag: Rename VNC 'VNC Password' field to opencanary.vnc.password
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_VNC'
+      field: 'opencanary.logdata.VNC Password'
+      target_field: opencanary.tftp.vnc.password
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename VNC 'VNC Server Challenge' field to opencanary.vnc.server_challenge
+      tag: Rename VNC 'VNC Server Challenge' field to opencanary.vnc.server_challenge
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_VNC'
+      field: 'opencanary.logdata.VNC Server Challenge'
+      target_field: opencanary.tftp.vnc.server_challenge
+      ignore_missing: true
+      ignore_failure: true
+
+  ################################
+  ## Parse Event Details to ECS ##
+  ################################
+  - append:
+      description: Append "event.category" according to event details
+      tag: Append "event.category" according to event details
+      field: event.category
+      value: 
+        - network
+        - intrusion_detection
+      allow_duplicates: false
+      ignore_failure: true
+  - date:
+      description: Set "event.created" from "opencanary.utc_time"
+      tag: Set "event.created" from "opencanary.utc_time"
+      field: opencanary.utc_time
+      target_field: event.created
+      formats:
+        - yyyy-MM-dd HH:mm:ss.SSSSSS
+      ignore_failure: true
```

```suggestion
  - set:
      description: Set "event.created" from "opencanary.utc_time"
      tag: set_event_created
      field: event.created
      copy_from: '@timestamp'
```
since we've already done this work.